Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -417,13 +417,30 @@ public LinearWeightedMovingAverage LWMA(string symbol, int period, Resolution? r
         public OnBalanceVolume OBV(string symbol, Resolution? resolution = null,
             Func<BaseData, TradeBar> selector = null)
         {
-            string name = CreateIndicatorName(symbol, "OBV", resolution);
+            var name = CreateIndicatorName(symbol, "OBV", resolution);
             var onBalanceVolume = new OnBalanceVolume(name);
-            RegisterIndicator(symbol, onBalanceVolume, resolution);
+            RegisterIndicator(symbol, onBalanceVolume, resolution, selector);
             return onBalanceVolume;
         }
 
         /// <summary>
+        /// Creates a new Average Directional Index indicator. 
+        /// The indicator will be automatically updated on the given resolution.
+        /// </summary>
+        /// <param name="symbol">The symbol whose Average Directional Index Average Directional Index we seek</param>
```

Comment:
looks like there's a little double speak double speak here :) - please remove the extra 'Average Direction Index'